src/lib/ng-v5/entry-point/resources/stylesheet.transform.ts

```diff
@@ -1,5 +1,6 @@
 import * as fs from 'fs-extra';
 import * as path from 'path';
+import stripBom = require('strip-bom');
```

Comment:
I rebased your changes to latest changes on master and these four lines (3x stylesheets, 1x template) are affected. Right?

Reply:
yes that's right, aslo in package.json replaced `'src/**/*.spec.ts'` with `\"src/**/*.spec.ts\"` in `test:specs` script as it breaks on windows

Reply:
Good. Then let's get in!